package.json

```
@@ -0,0 +1,41 @@
+{
+  "name": "cache-manager",
+  "version": "1.0.0",
+  "description": "paquete manager para el manejo de cache",
+  "main": "index.js",
+  "scripts": {
+    "test": "export TEST_ENV=true; mocha --exit -R nyan --recursive tests/",
+    "watch-test": "export TEST_ENV=true; mocha --exit -R nyan -w --recursive tests/",
+    "coverage": "nyc npm test",
+    "test-ci": "nyc --reporter=html --reporter=text mocha --recursive tests/",
+    "start": "node develop.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/janis-commerce/cache-manager.git"
+  },
+  "keywords": [],
+  "author": "",
```

```suggestion
  "author": "Janis",
```
El autor del package debe ser "Janis"